Sortable.js

```diff
@@ -701,7 +701,6 @@
 				canSort = options.sort;
 
 			if (evt.preventDefault !== void 0) {
-				evt.preventDefault();
```

Comment:
Why? What a reason for this changes? Because of this changes, you break backward compatibility.

Reply:
Because `preventDefault()` will throw an error numerous times when `passive` is set to `true` for `captureMode`. I guess we can get around this by checking if the browser supports `passive`.

Reply:
Clear, then need don't remove this a call, but add an additional condition, ex: `!passiveMode`.
`passiveMode` — must be defined here: https://github.com/RubaXa/Sortable/pull/1202/files#diff-a8f06fcf61e5b94cb2e81ac878a55678R1487

Let me correct now and you check, ok?